include/dali/core/geom/mat.h

```diff
@@ -22,7 +22,7 @@
 #include "dali/core/geom/vec.h"
 
 #ifndef MAT_LAYOUT_ROW_MAJOR
-#define MAT_LAYOUT_ROW_MAJOR 0
+#define MAT_LAYOUT_ROW_MAJOR 1
```

Comment:
With row-major layout we can pass matrices in more natural layout directly to kernels.